debian/daos.install

```diff
@@ -1,8 +1,7 @@
-usr/lib64/libdaos_common.so
 etc/daos/memcheck-cart.supp
-# TODO: this should move from daos_srv to daos
```

Comment:
Should this TODO have moved also, or is this no longer a TODO?

Reply:
It's no longer a todo.